docs/api.rst

```diff
@@ -5020,17 +5020,17 @@ Exporting History Events
    :statuscode 409: No user is logged in or failure at event export.
    :statuscode 500: Internal rotki error
 
-.. http:patch:: /api/(version)/history/events/export
+.. http:put:: /api/(version)/history/events/export
```

Comment:
This is a correct change but there is one more thing off. The name of the function here should be `put`:

https://github.com/rotki/rotki/blob/2b358c20a052729238f79c24733d88bc8448e6d9/rotkehlchen/api/v1/resources.py#L2971

Please do it since you are changing this here in the docs